src/Rule/Uuid.php

```diff
@@ -23,17 +21,41 @@ class Uuid extends Regex
     const INVALID_UUID = 'Uuid::INVALID_UUID';
 
     /**
-     * UUID Version 4.
+     * UUID NIL & version binary masks
      */
-    const UUID_V4 = 4;
+    const UUID_NIL = 0b000001;
+    const UUID_V1  = 0b000010;
+    const UUID_V2  = 0b000100;
+    const UUID_V3  = 0b001000;
+    const UUID_V4  = 0b010000;
```

Comment:
This will break backwards compatibility (I know we use it in some projects), could you make it so that you prove an int in the constructor, which gets mapped to this?